front/pages/auth/signup/success.vue

```diff
@@ -0,0 +1,232 @@
+<template>
+  <div id="signupSuccessPage">
+    <v-layout row class="gridContainer">
+      <v-flex class="lefgImgContainer" hidden-sm-and-down>
+        <img
+          class="imgPersonWithPhone"
+          src="~/assets/images/person_with_phone.png"
+        />
+      </v-flex>
+
+      <!-- Register Div -->
+      <v-flex class="successMessage">
+        <!-- Header Images -->
+        <img class="logoIcon" src="~/assets/images/signup/dark_gray_lamp.png" />
+        <br />
+        <img class="logoText" src="~/assets/images/logo_text.png" />
+
+        <!-- Hero Message -->
+        <div class="heroMsg">
+          Yay!
+        </div>
+
+        <!-- Hero Description -->
+        <div class="heroDescription">
+          Welcome to Daily Idea!<br />
+          We've sent you an email confirmation link
+        </div>
+
+        <!-- Continue Button -->
+        <v-btn large class="okBtn">OK</v-btn>
+
+        <!-- Social Login Icons -->
+        <div class="socialIconContainer">
+          <v-btn small outline fab color="primary">
+            <v-icon>fab fa-facebook-f</v-icon>
+          </v-btn>
+          <v-btn small outline fab color="primary">
+            <v-icon>fab fa-twitter</v-icon>
+          </v-btn>
+          <v-btn small outline fab color="primary">
+            <v-icon>fab fa-google-plus-g</v-icon>
+          </v-btn>
+        </div>
+      </v-flex>
+
+      <v-flex class="rightImgContainer" hidden-sm-and-down>
+        <img
+          class="imgLightGrayLamp"
+          src="~/assets/images/signup/light_gray_lamp.png"
+        />
+        <img
+          class="imgPersonWithPhone"
+          src="~/assets/images/signup/lady_with_phone.png"
+        />
+      </v-flex>
+    </v-layout>
+
+    <!-- Fixed Footer -->
+    <v-layout
+      hidden-sm-and-down
+      class="fixedFooter"
+      :style="{
+        'background-image':
+          'url(' + require('~/assets/images/signup/footer_background.png') + ')'
```

Comment:
why inline and not in class?

Reply:
Actually, I could not find solution to render images in scss from assets. So I have to add this through markup. Is there a way to use assets from scss, I will use them. Let me know.